synapse/storage/schema/delta/55/add_spans_to_device_lists.sql

```diff
@@ -0,0 +1,16 @@
+/* Copyright 2019 The Matrix.org Foundation C.I.C.d
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ALTER TABLE device_lists_outbound_pokes ADD context TEXT;
```

Comment:
typically we include a comment explaining what the new column means.

Reply:
please can we have a comment?

Reply:
How's that?

Reply:
it's a bit wordy, to be honest :/. I mean, it's great, but I think it's more detail than we need here.

> Opentracing context data for inclusion in the `device_list_update` EDUs, as a json-encoded dictionary

Reply:
Fair enough